segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -314,7 +314,8 @@ public static void updateCoreAttributes(Map<Integer, String> backUpMetadataSegme
                                             Duration timeout) throws InterruptedException, ExecutionException,
             TimeoutException {
         Preconditions.checkState(backUpMetadataSegments.size() == containersMap.size(), "The number of " +
-                "back-up metadata segments and containers should match.");
+                "back-up metadata segments = " + backUpMetadataSegments.size() + " and number of containers = "
```

Comment:
you can use % s like below
Preconditions.checkState("The number of back-up metadata segments = %s and number of containers = %s "should match.", backUpMetadataSegments.size(), containersMap.size())

Reply:
Fixed.